src/main/java/edu/njit/jerse/ashe/services/SpeciminTool.java

```diff
@@ -40,14 +45,14 @@ private SpeciminTool() {
     }
 
     /**
-     * Executes and manages the Specimin tool using the specified paths and targets.
+     * Executes and manages the {@code Specimin} tool using the specified paths and targets.
      *
-     * @param root         The root directory for the tool.
-     * @param targetFile   File to be targeted by the tool.
-     * @param targetMethod Method to be targeted by the tool.
-     * @return The directory path where the minimized file is saved.
-     * @throws IOException          If there's an error executing the command or writing the minimized file.
-     * @throws InterruptedException If the process execution is interrupted.
+     * @param root         the root directory for the tool
+     * @param targetFile   file to be targeted by the tool
+     * @param targetMethod method to be targeted by the tool
+     * @return the directory path where the minimized file is saved
+     * @throws IOException          if there's an error executing the command or writing the minimized file
+     * @throws InterruptedException if the process execution is interrupted
```

Comment:
These are good changes, thanks.
It is better to make a separate pull request for each separate concern.  So, these would be better in a separate pull request.